### PR TITLE
More edit controls

### PR DESCRIPTION
**Pull request by dabreegster**
#240 

https://user-images.githubusercontent.com/1664407/200313141-793b6aff-1e2f-4bf3-9048-4be609614850.mp4

1) A (+) button to insert a lane at a particular position
2) Add missing card for sidewalks!
3) Click direction arrow to toggle it

A next possible step is clicking the lane type icon to get a little modal popup and choose the type. So then inserting a lane and changing the type would become much more fluid

### CODE REVIEW

File: web_editor/main.css [hunk 1]
```diff
@@ -33,6 +35,13 @@
   width: 50px;
 }
 
+.insert-lane {
+  position: absolute;
+  right: -15px;
```
Comment (dabreegster) [inline web_editor/main.css:40]: Screenshot from 2022-11-07 11-50-38
Actually, I got the icons to be in a better place. Is this a reasonable way of achieving it?

Comment (BudgieInWA) [reply web_editor/main.css:40]: Perfect use of `position: absolute`! We want to anchor it to the bottom right of the card and that's idiomatic CSS to do so.

File: web_editor/js/cards.js [hunk 2]
```diff
@@ -11,7 +11,8 @@ export function makeLaneCard(lane, idx, app) {
   }
   node.appendChild(wrapInCenterDiv(width(lane)));
 
-  var finalRow = document.createElement("div");
+  var editRow = document.createElement("div");
+  editRow.align = "center";
```
Comment (BudgieInWA) [inline web_editor/js/cards.js:15]: I'd prefer this done in CSS. `text-align: center` (for centering inline children of a block element) or `margin: 0 auto` (for centering a block element within its parent) will cover most cases.

**Review by BudgieInWA — APPROVED** [review]
:+1:

Using a "dropdown" type interface for changing lane type as you describe sounds good.

File: web_editor/js/cards.js (hunk 2 above)
Comment (dabreegster) [reply web_editor/js/cards.js]: Done!

**Pull request merged**